packages/common/src/components/CopyableToClipboard/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Clipboard from 'clipboard';
+import { invokeGlobalErrorHandler } from '../../utilities/splash.screen';
+
+interface IProps {
+  textGetter: () => string;
+  globallyUniqueSelector?: string;
```

Comment:
in makercore, I've started adding jsdoc comments for each of the properties of an interface especially when they aren't obvious. I feel like this might be a good candidate

Reply:
Specifically relating to its implicit connection with idIfAny